asset/css/markbind.css

```diff
@@ -40,3 +40,26 @@ blockquote {
         background-color: #fff;
     }
 }
+
+html, body {
+    min-height: 100vh;
+}
+
+#app.container-fluid {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
+
+#flexDiv{
+    flex: 1 0 auto;
+}
+
+footer {
+    color: black;
+    background-color: #ebebeb;
```

Comment:
If we are keeping the colors, remember to sort the attributes according to alphabetical order.